repository-module/src/main/java/com/gmail/zagurskaya/online/cash/repository/properties/DatabaseProperties.java

```diff
@@ -0,0 +1,40 @@
+package com.gmail.zagurskaya.online.cash.repository.properties;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component("databaseProperties")
+public class DatabaseProperties {
```

Comment:
We need to use configuration of spring boot with this properties. Need to remove class.